### PR TITLE
fix(ui): broken property full-URI hover

### LINKED ISSUE #420: Fix wrong value in hover on properties
Fix the wrongly concatenated hover in the case of an external property:
Screenshot 2021-01-18 at 16 21 40

### PR DESCRIPTION
**Pull request by martinmaillard**
Fix #420

### CODE REVIEW

File: ui/src/components/PropertyDisplay.vue [hunk 1]
```diff
@@ -23,7 +23,9 @@ export default class PropertyDisplay extends Vue {
   }
 
   get expanded (): string {
-    return this.createIdentifier(this.value)
+    return this.value.includes(':')
+      ? expand(this.value)
+      : this.createIdentifier(this.value)
```
Comment (tpluscode) [inline ui/src/components/PropertyDisplay.vue:28]: The underlying `Organization#createIdentifier` already does that but it treats strings a term template.

I would also pass the `Term` instead. (needs also update a type declarations)

```suggestion
    return this.createIdentifier(this.term)
```

Comment (martinmaillard) [reply ui/src/components/PropertyDisplay.vue:28]: I don't see any call to `expand` in `createIdentifier`

Comment (tpluscode) [reply ui/src/components/PropertyDisplay.vue:28]: True, it does not expand. The input is either a Term (returned as-is) or a template, such as `cc:propertyTemplate` which gets concatenated. Prefixed URI is not an expected input

Comment (martinmaillard) [reply ui/src/components/PropertyDisplay.vue]: Changed

Comment (martinmaillard) [reply ui/src/components/PropertyDisplay.vue]: Crap, I need to change again then :D I wonder why it worked

**Pull request merged**